test/unit/services/test-get-caseload.js

```diff
@@ -24,7 +25,12 @@ const CASELOAD = {
   a: 0
 }
 
```

Comment:
Could you include multiple workload owners' caseloads of each type? Just to make sure queries / functions work for larger data sets?